arex-storage-web-api/src/main/java/com/arextest/storage/beans/StorageAutoConfiguration.java

```diff
@@ -46,7 +57,18 @@ public StorageAutoConfiguration(StorageConfigurationProperties configurationProp
     @Bean
     @ConditionalOnMissingBean(MongoDatabase.class)
     public MongoDatabase mongoDatabase() {
-        return MongoDbUtils.create(properties.getMongodbUri());
+        MongoDatabase database = MongoDbUtils.create(properties.getMongodbUri());
+        for (MockCategoryType category : MockCategoryType.DEFAULTS) {
+            setTTLIndex(category, database);
+        }
+        return database;
+    }
+
+    private void setTTLIndex(MockCategoryType category, MongoDatabase mongoDatabase) {
+        String categoryName = ProviderNames.DEFAULT + category.getName() + COLLECTION_PREFIX;
+        MongoCollection<AREXMocker> collection = mongoDatabase.getCollection(categoryName, AREXMocker.class);
+        collection.createIndex(new Document(EXPIRATION_TIME_COLUMN_NAME, 1),
```

Comment:
Does ConfigFileMocker need to have an expiration?

Reply:
yes, 40days.